packages/@n8n/db/src/entities/workflow-dependency-entity.ts

```diff
@@ -0,0 +1,57 @@
+import { Column, Entity, Index, JoinColumn, ManyToOne } from '@n8n/typeorm';
+
+import { WithTimestampsAndStringId } from './abstract-entity';
+import type { WorkflowEntity } from './workflow-entity';
+
+export type DependencyType = 'credential' | 'nodeType' | 'webhookPath' | 'workflowCall';
+
+@Entity()
```

Comment:
nit: Could also include the table name

```suggestion
@Entity({ name: 'workflow_dependency' })
```